src/meta/test/ProcessorTest.cpp

```diff
@@ -41,8 +41,12 @@ using nebula::cpp2::SupportedType;
 using apache::thrift::FragileConstructor::FRAGILE;
 
 TEST(ProcessorTest, AddHostsTest) {
+    auto workers = std::make_shared<thread::GenericThreadPool>();
```

Comment:
You could move the workers and ioPool related code into TestUtils::initKV

Reply:
Aha, good idea 👍